tests/test_workflow.py

```diff
@@ -156,6 +158,7 @@ def test_get_workflow_tasks_normal_and_basic(workflow_mock):
     assert tasks[0] == {
         "id": "c0d04ec3-98d7-4183-902f-5bcb2a176d89",
         "name": "sobloo-s2-l1c-aoiclipped:1",
+        "blockVersionTag": "2.2.2",
```

Comment:
Cool! Does the API now return the version tag too? Or did it always do that?

Reply:
Always returned it, now just added as default output to `basic=True` as well so needed to add to mock test.